src/catalogue/category/catalogueCategoryDialog.component.tsx

```diff
@@ -490,7 +490,15 @@ const CatalogueCategoryDialog = React.memo(
     ]);
 
     return (
-      <Dialog open={open} onClose={handleClose} maxWidth="lg" fullWidth>
+      <Dialog
+        open={open}
+        onClose={(_event: object, reason: string) =>
```

Comment:
Could we explore alternative implementations? Much of the existing logic appears redundant now.